pkg/registry/registry.go

```diff
@@ -111,7 +111,10 @@ func (r *Registry) ListImageTags(image string) ([]string, error) {
 	var tags []string
 	o := func() error {
 		imageTags, err := r.registryClient.Tags(images.Name(r.organisation, image))
-		if err != nil {
+		if IsRepositoryNotFound(err) {
```

Comment:
Very cool if that works - can you share which settings you set to make this work? In quay for example?

Reply:
There was also a public user who asked for support of this behavior. 
https://github.com/giantswarm/retagger/issues/263

Reply:
in aliyun, there is specific settings on resource `namespace`  (similar to organisation in quay` that is called `auto-create repository` so i checked that out

in quay you dont need extra box checked but the account need to have admin privileges so it can create repositories

That is basically all that needs to be done. It just needed few tries to get it straight.

Reply:
I am not sure if it makes sense to document this somewhere since this seem very provider specific and any new registyr will probabbly have different settings

Reply:
Answer is totally sufficient to me. Makes sense :+1:

Thanks for taking care of this :tada: